.travis.yml

```diff
@@ -1,9 +1,7 @@
 ---
-dist: bionic
 language: python
-python: "3.7"
+python: "3.9"
 sudo: required
-
 services:
   - docker
 install:
```

Comment:
Suggestion: to change `pip install pipenv` to add `-I` option `pip install -I pipenv` [See](https://github.com/idealista/cookiecutter-ansible-role/issues/95)